geanyvc/src/Makefile.rel.am

```diff
@@ -0,0 +1,35 @@
+if ENABLE_GEANYVC
+geanyplugins_LTLIBRARIES += %D%/geanyvc.la
+
+%C%_geanyvc_la_SOURCES = \
+	%D%/externdiff.c \
+	%D%/geanyvc.c \
+	%D%/utils.c \
+	%D%/vc_bzr.c \
+	%D%/vc_cvs.c \
+	%D%/vc_git.c \
+	%D%/vc_hg.c \
+	%D%/vc_svk.c \
+	%D%/vc_svn.c \
+	%D%/geanyvc.h
+
+%C%_geanyvc_la_CFLAGS = \
+	$(AM_CFLAGS) \
+	$(GTKSPELL_CFLAGS)
+
+%C%_geanyvc_la_LIBADD = \
+	$(GTKSPELL_LIBS) \
+	$(COMMONLIBS)
+
+endif
+
+if UNITTESTS
+TESTS += %D%/unittests
+check_PROGRAMS += %D%/unittests
+%C%_unittests_SOURCES = %D%/unittests.c %D%/utils.c
+%C%_unittests_CFLAGS  = $(GEANY_CFLAGS) -DUNITTESTS
+%C%_unittests_LDADD   = @GEANY_LIBS@ $(INTLLIBS) @CHECK_LIBS@
```

Comment:
…or not, this one has it, my bad.  Makes me believe even more it was missing in previous location :)